build/commands/scripts/sync.js

```diff
@@ -60,25 +58,15 @@ async function RunCommand() {
 
   config.update(program)
 
-  if (program.ignore_chromium) {
-    Log.warn(
-        '--ignore_chromium is deprecated, please replace with ' +
-        '--sync_chromium=false')
-    program.sync_chromium = false
-  }
-
   depotTools.installDepotTools()
 
-  if (program.init || !fs.existsSync(config.defaultGClientFile)) {
+  if (
+    program.init ||
+    program.target_os ||
```

Comment:
this is problematic because I always want to have `target_os = ["ios"]` in my `.gclient` file so I can build for mac or ios, but if I set `target_os` in `.env` then it will also override for my builds. It's also confusing that this `target_os` can take a list and `target_os` in other commands cannot.